### ISSUE TITLE
Automatic non-regression testing for a11y

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/Orange-Boosted-Bootstrap#1350]
The goal of this issue is to put in place an automatic non-regression testing tool for a11y.

* Analyze each page of the documentation like pa11y can do and detect automatically that some basic rules of a11y are applied.
  * Example: a modal use `aria-modal`
* For JS component, it is sometimes already done but should be generalized in the unit tests

/CC @Aniort
